docs/data/base/components/checkbox/UnstyledCheckboxIntroduction/tailwind/index.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
-import {
-  Checkbox as BaseCheckbox,
-  type CheckboxIndicatorProps,
-  type CheckboxProps,
+import * as BaseCheckbox from '@base_ui/react/Checkbox';
+import type {
+  CheckboxIndicatorProps,
+  CheckboxRootProps,
 } from '@base_ui/react/Checkbox';
```

Comment:
I'm not convinced about having to import twice from the same path. How do you feel about removing the `Checkbox` prefix from types as well and importing them from the namespace (for example `Checkbox.RootProps`)?